docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -431,15 +433,15 @@ for i, t in enumerate(df_retweet['tweet_id']):
 
 When running this loop, you are likely to see this:
 
-```python
+```{ .yaml .no-copy }
```

Comment:
python no copy

Reply:
I think you're looking at outdated file but I changed it to output since this is what a user sees